package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {},
   "private": true,
   "dependencies": {
+    "@chatkitty/core": "^1.2.13",
```

Comment:
The type definitions should be imported from https://www.npmjs.com/package/chatkitty-platform-sdk, which defines server-side resources. @chatkitty/core is client-side and has stuff like relays and WebSockets, which aren't relevant for server-side plugins.